source-academy/cadet-frontend
# Fix issue with WorkspaceSaga tests

### Description

Event handling in WorkspaceSaga no longer results in console error when running tests.
The problem was that any explicit reference to the store caused some error to be thrown, fixed by using a saga instead of a normal function and putting an action instead.

### Type of change


### How to test

Double check that the test doesn't have any errors. Also might be worth checking that events still get registered properly, and that events don't register when not logged in.

### Checklist
